sql-plugin/src/main/scala/org/apache/spark/sql/hive/rapids/HiveProviderImpl.scala

```diff
@@ -145,41 +145,47 @@ class HiveProviderImpl extends HiveProvider {
             val ctrlASeparatedFormat = "1" // Implying '^A' field delimiter.
             val lineDelimiterKey     = "line.delim"
+            val escapeDelimiterKey   = "escape.delim"
             val newLine              = "\n"
 
             if (storage.inputFormat.getOrElse("") != textInputFormat) {
-              willNotWorkOnGpu(s"Unsupported input-format found: ${storage.inputFormat}. " +
-                s"Only $textInputFormat is currently supported.")
+              willNotWorkOnGpu(s"unsupported input-format found: ${storage.inputFormat}, " +
+                s"only $textInputFormat is currently supported")
             }
 
-            if(storage.serde.getOrElse("") != lazySimpleSerDe) {
-              willNotWorkOnGpu(s"Unsupported serde found: ${storage.serde}. " +
-                s"Only $lazySimpleSerDe is currently supported.")
+            if (storage.serde.getOrElse("") != lazySimpleSerDe) {
+              willNotWorkOnGpu(s"unsupported serde found: ${storage.serde}, " +
+                s"only $lazySimpleSerDe is currently supported")
             }
 
-            if(storage.properties.getOrElse(serializationKey, "") != ctrlASeparatedFormat) {
-              willNotWorkOnGpu(s"Unsupported serialization format found: " +
-                s"${storage.properties.getOrElse(serializationKey, "")}. " +
-                s"Only \'^A\' separated text input (i.e. serialization.format=1) " +
-                s"is currently supported.")
+            val serializationFormat = storage.properties.getOrElse(serializationKey, "")
+            if (serializationFormat != ctrlASeparatedFormat) {
+              willNotWorkOnGpu(s"unsupported serialization format found: " +
+                s"$serializationFormat, " +
+                s"only \'^A\' separated text input (i.e. serialization.format=1) " +
+                s"is currently supported")
             }
 
             val lineTerminator = storage.properties.getOrElse(lineDelimiterKey, newLine)
-            if(lineTerminator != newLine) {
-              willNotWorkOnGpu(s"Unsupported line terminator found: " +
-                s"$lineTerminator. " +
-                s"Only newline (\\n) separated text input  is currently supported.")
+            if (lineTerminator != newLine) {
+              willNotWorkOnGpu(s"unsupported line terminator found: " +
+                s"$lineTerminator, " +
+                s"only newline (\\n) separated text input  is currently supported")
+            }
+
+            if (!storage.properties.getOrElse(escapeDelimiterKey, "").equals("")) {
+              willNotWorkOnGpu("escapes are not currently supported")
             }
           }
 
           private def checkIfEnabled(): Unit = {
             if (!conf.isHiveDelimitedTextEnabled) {
-              willNotWorkOnGpu("Hive Text I/O has been disabled. To enable this, " +
+              willNotWorkOnGpu("hive text I/O has been disabled. To enable this, " +
```

Comment:
Nit: Hive is a proper name, should be capitalized.
```suggestion
              willNotWorkOnGpu("Hive text I/O has been disabled. To enable this, " +
```